frontend/components/CommentRating.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from "react";
+import { StyleSheet, View, TextInput } from "react-native";
+
+interface StarRateProps {
```

Comment:
nit but can we rename this? it made me think this is the star component but it's just the comment text
```suggestion
interface CommentRatingProps {
```